README.md

```diff
@@ -53,6 +53,15 @@ end
 Everypolitician::Index.new.all_legislatures.each do |legislature|
   puts "#{legislature.name} in #{legislature.country.name} has #{legislature.person_count} member(s)"
 end
+
+# Fetch Everypolitician::Popolo::LegislativePeriod for the latest term
+# Note that this may involve a **substantial download**
+assembly = Everypolitician.country(slug: 'Albania').legislature(slug: 'Assembly')
+term_details = assembly.latest_term
+puts 'Members of the Assembly:'
+term_details.memberships.each do |m|
+  puts m.person.name
+end
```

Comment:
So, I wanted a nice simple example here and I'm afraid that making people unique isn't going to be:

```
term_details.memberships.map(&:person).uniq(&:id).each do |p|
  puts p.name
end
```